### PR TITLE
Selective ci

### PR DESCRIPTION
**Pull request by PavelLinearB**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=RD0s9tK)

image

image

image

### CODE REVIEW

File: docs/execution-model.md [hunk 1]
```diff
@@ -8,8 +8,62 @@ When a central `cm` repository is set with the CI/CD runner, the events for PRs
...
+	- commit
+
+automations:
+  assign_code_experts:
+	if: 
+	  - true
+	run:
+	  - action: add-reviewers@v1
+		args:
+		  reviewers: {{ repo | codeExperts(gt=10) }}
```
Comment (vim-zz) [inline docs/execution-model.md:51]: the example use uneven spaces for indendation, use 2 spaces for every indentation level

File: docs/execution-model.md [hunk 2]
```diff
@@ -8,8 +8,62 @@ When a central `cm` repository is set with the CI/CD runner, the events for PRs
...
+```
+
+-  Explain code experts only if the label “suggest-reviewer” exists.  
+    After each commit **or** after adding the label.
```
Comment (vim-zz) [inline docs/execution-model.md:55]: some description of what will happen once you use the `on: commit` compared to not using it, would be very helpful.

File: docs/execution-model.md [hunk 3]
```diff
@@ -8,8 +8,62 @@ When a central `cm` repository is set with the CI/CD runner, the events for PRs
...
+gitStream supports the following explicit triggers:
+
+- `merge`: Trigger when merging the PR
+- `pr_created`: Trigger when the PR is created
+- `commit`: Trigger on each commit
+- `comment_added`: Trigger on each added comment
+- `label_added`: Trigger on each added label
+- `label_removed`: Trigger on removed label
+
+Explicit triggers can be used at the file level, specific to each automation separately, or a combination of the two. In case triggers are listed at the file level **and** specific automation, the automation will be triggered according to both triggers.
```
Comment (vim-zz) [inline docs/execution-model.md:35]: Worth mentioing the fact triggers are set per automation block, as either default (implicit), union of in-autoamtion + in-cm (explicit)

**Review by vim-zz — CHANGES_REQUESTED** [review]

**Comment by PavelLinearB** [comment]
#jit_bypass_commit

**Review by vim-zz — CHANGES_REQUESTED** [review]

**Review by vim-zz — APPROVED** [review]

**Pull request merged**